src/main/java/Duke.java

```diff
@@ -1,10 +1,264 @@
+import src.main.java.*;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+import static java.lang.Integer.parseInt;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+    int index;
+    List<Task> ls;
+
+    public Duke() {
+        this.index = 0;
+        this.ls = new ArrayList<>();
+    }
+
+    // prints introduction of klaun
+    public void introDuke() {
+        System.out.println("I'm Klaun (=^.^=) How are you doing today ?\n");
+        System.out.println("Is there anything I can help you with ?\n");
+        System.out.println("<------------------------------------------------------------>\n");
+    }
+
+    public int getIndex() {
+        return this.index;
+    }
+
+    // prints output based on command
+    public void sayBye() {
+        System.out.println("++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++\n");
+        System.out.println("oh man ... bye ~~ o.o \n");
+        System.out.println("++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++\n");
+    }
+
+    // prints list of tasks
+    public void getList() {
+        System.out.println("##############################################################\n");
+        System.out.println("Here's your amazing task list :");
+        for (int i = 0; i < this.index; i++) {
+            Task task = this.ls.get(i);
+            String item = task.getItem();
+            String status = task.getStatus();
+            String taskType = task.getSign();
+            int num = i + 1;
+
+            if (task instanceof Todo) { // if task is to-do
```

Comment:
could perhaps overwrite the toString() method such that polymorphism can be utilised, rather than performing multiple if else checks!